build.gradle

```diff
@@ -13,7 +13,7 @@ apply plugin: 'osgi'
 def getDevelopmentVersion() {
     def output = new StringBuilder()
     def error = new StringBuilder()
-    def gitShortHash = "git -C ${projectDir} rev-parse --short HEAD".execute()
+    def gitShortHash = """git rev-parse --short HEAD""".execute()
```

Comment:
Not sure about this change.  Some one else wanted this in for Netbeans something something.  Did you mean for this to be changed??